### PR TITLE
Default softWrap

### PR DESCRIPTION
**Pull request by Aerijo**
Here's the pull. I only enabled it when the `text.tex.latex` grammar is used, so it should work for all normal LaTeX files. 

It should not set softwrapping to true in the log or bib files, as I believe them to be closer to a structured programming setup, where personal preference may go either way. If you want this changed (for consistency) I'll do it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.1.2
```
Comment (yudai-nkt) [inline CHANGELOG.md]: Remove this line. I don't think this is a patch release. Version number should be written just before a new release.

**Comment by yudai-nkt** [comment]
Also, please resolve conflicts.

Requesting contributors to update CHANGELOG.md tends to cause a merge conflict. Should we stop this?

**Comment by yudai-nkt** [comment]
Note: Fix #18.

**Comment by Aerijo** [comment]
It seems that changing CHANGELOG.md after someone forks this package leads to the conflicts. 
I don't know the best way to deal with it, but fixing the conflict amounts to deleting the 'conflict syntax' that git adds, so that both your lines and the conflicting lines get added.

**Pull request merged**